api/src/main/java/org/openmrs/module/referencemetadata/reporting/reports/ListOfNewPatientRegistrations.java

```diff
@@ -72,7 +72,7 @@ public ReportDefinition constructReportDefinition() {
 	@Override
 	public List<ReportDesign> constructReportDesigns(ReportDefinition reportDefinition) {
 		List<ReportDesign> l = new ArrayList<ReportDesign>();
-		l.add(ReportManagerUtil.createExcelDesign("283638f8-487b-11e7-a919-92ebcb67fe33", reportDefinition));
+		l.add(ReportManagerUtil.createExcelDesign("c1e93868-633a-11e7-907b-a6006ad3dba0", reportDefinition));
```

Comment:
Could you please create truly random uuids e.g. using https://www.uuidgenerator.net/? They differ only a few chars from ListOfDiagnosis and ListOfProviders...